app/src/main/kotlin/dev/soupslurpr/appverifier/InternalVerificationInfoDatabase.kt

```diff
@@ -1900,4 +1900,18 @@ val internalVerificationInfoDatabase = setOf(
             )
         )
     ),
+    InternalDatabaseVerificationInfo(
+        "com.logseq.app",
+        listOf(
+            Hashes(
+                listOf(
+                    Source.GITHUB,
```

Comment:
```suggestion
                    Source.WEBSITE,
                    Source.GITHUB,
```

When visiting the official website logseq.com with an Android device it'll offer a direct APK download with the same fingerprint as the GitHub releases.